mmv1/products/workstations/WorkstationCluster.yaml

```
@@ -60,6 +60,14 @@ examples:
     min_version: 'beta'
     vars:
       workstation_cluster_name: 'workstation-cluster-custom-domain'
+  - name: 'workstation_cluster_tags'
+    primary_resource_id: 'default'
+    min_version: 'beta'
+    test_env_vars:
```

Somehow this lost the other vars:

```suggestion
    vars:
      workstation_cluster_name: 'workstation-cluster-tags'
      key_short_name: 'keyname'
      value_short_name: 'valuename'
    test_env_vars:
```